app/models/user.js

```diff
@@ -43,6 +43,7 @@ export default EmberObject.extend({
 		});
 
 		return fetch(`${config.helios.internalUrl}${queryString}`, {
+			headers: {'X-Wikia-Internal-Request': '1'},
```

Comment:
I just wonder if we shouldn't use this like this:
```js
const headers = new Headers({'X-Wikia-Internal-Request': '1'});
```

and then supply it to the `fetch()`